adam_core/dynamics/propagation.py

```diff
@@ -163,7 +163,7 @@ def propagate_2body(
             vy=orbits_propagated[:, 4],
             vz=orbits_propagated[:, 5],
             covariance=cartesian_covariances,
-            time=Times.from_astropy(
+            time=Timestamp.from_astropy(
                 Time(t1_, scale="tdb", format="mjd"),
             ),
```

Comment:
There are a _lot_ of `Timestamp.from_astropy(astropy.Time(...))` calls littered throughout the codebase. Those should be shrinkable down to just `Timestamp(...)` in some way - possibly a new direct constructor. For example here it would be `Timestamp.from_mjd(t1_, scale="tdb"). 

I could do those now, if that seems useful.

Reply:
Yes, definitely.